packages/ui/components/profile/ProfileVoteCard.tsx

```diff
@@ -1,5 +1,74 @@
-export interface ProfileVoteCardProps {}
+import clsx from 'clsx'
+import { useTranslation } from 'react-i18next'
 
-export const ProfileVoteCard = ({}: ProfileVoteCardProps) => {
-  return <p>placeholder</p>
+import { Button } from '../Button'
+import { ProfileCardWrapper } from './ProfileCardWrapper'
+import { ProfileVoteButton } from './ProfileVoteButton'
+
+export interface ProfileVoteCardProps {
+  variant: 'abstain' | 'noWithVeto' | 'no' | 'yes' | 'unselected'
+  loading?: boolean
+  votingPower: number
+  daoName: string
+  walletName: string
+  profileImgUrl: string
+}
+
+export const ProfileVoteCard = ({
+  variant,
+  loading,
+  votingPower,
+  daoName,
+  walletName,
+  profileImgUrl,
+}: ProfileVoteCardProps) => {
+  const { t } = useTranslation()
+
+  return (
+    <ProfileCardWrapper
+      compact
+      daoName={daoName}
+      imgUrl={profileImgUrl}
+      isMember
+      walletName={walletName}
+    >
+      <div className="flex flex-row justify-between items-center secondary-text">
+        <p>{t('title.votingPower')}</p>
+        <p className="font-mono text-text-primary">{votingPower}%</p>
```

Comment:
Can we wrap this in `formatPercentOf100` instead of adding the percent here? It also does some fraction limiting stuff.